Dockerfile

```diff
@@ -1,17 +1,23 @@
-
 FROM ruby:2.4.1
 MAINTAINER Conor Heine <conor.heine@gmail.com>
```

Comment:
What is the `MAINTAINER` field used for? (not familiar with Docker)
The `Dockerfile` was originally added in https://github.com/Thibaut/devdocs/pull/303. At this point we should probably change it to my name/email, or remove it.